package.json

```diff
@@ -78,10 +78,16 @@
           "description":
             "Specifies which tool or tools will be used to get diagnostics. If you choose both \"merlin\" and \"bsb\", merlin will be used while editing and bsb when saving."
         },
+        "reason.format.ocamltool": {
+          "enum": ["ocp-indent", "ocamlformat"],
+          "default": "ocamlformat",
```

Comment:
Hmm, setting the default to `ocamlformat` will probably break a lot of users who get upgraded and don't have it installed on their system. We should probably leave this as `ocp-indent`?